### PR TITLE
[t787] Media scrape/export fixes

### PR DESCRIPTION
**Pull request by secretrobotron**
1. For media obejcts, use data-butter-source attribute, then revert to currentSrc attrib for html5 media only
2. Copy attributes, class, id individually instead of recreating style and attributes through verbose methods
3. Tests

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Invalid

Command not implemented: ``.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/f38af67688d7d0b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/f38af67688d7d0b/output.txt

Total script time: 0.32 mins

Checking 19b0812c5e24f2825023a82990ad86de25e436d8 of pull request #267

#### Published

You can view your repo files at: http://butyr.org:8000/f38af67688d7d0b

File: src/core/popcorn-wrapper.js [hunk 1]
```diff
@@ -202,11 +202,22 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
...
           var parentNode = targetElement.parentNode,
-              newElement = document.createElement( "div" );
+              newElement = document.createElement( "div" ),
+              attributes;
 
           newElement.id = targetElement.id;
-          newElement.style.cssText = getComputedStyle( targetElement ).cssText;
+          attributes = targetElement.attributes;
+          if( attributes ){
+            for(var i = attributes.length - 1; i >= 0; i-- ) {
```
Comment (ScottDowne) [inline src/core/popcorn-wrapper.js]: Missing a tiny space before the var.

File: src/main.js [hunk 2]
```diff
@@ -519,11 +519,15 @@
...
-            if( ["VIDEO", "AUDIO" ].indexOf( medias[ i ].nodeName ) > -1 ) {
-              url = medias[ i ].currentSrc;
-            } else {
-              url = medias[ i ].getAttribute( "data-butter-source" );
+            mediaObj = medias[ i ];
+
+            if( mediaObj.getAttribute( "data-butter-source" ) ){
+              url = mediaObj.getAttribute( "data-butter-source" );
+            }
+            else if( ["VIDEO", "AUDIO" ].indexOf( mediaObj.nodeName ) > -1 ) {
```
Comment (ScottDowne) [inline src/main.js]: another tiny space just before the "VIDEO"

**Pull request merged**